lib/rambulance/test_helper.rb

```diff
@@ -28,7 +28,7 @@ def with_exceptions_app(enabled: true)
       org_show_exceptions          = Rails.application.env_config['action_dispatch.show_exceptions']
 
       Rails.application.env_config['action_dispatch.show_detailed_exceptions'] = !enabled
-      Rails.application.env_config['action_dispatch.show_exceptions']          = enabled
+      Rails.application.env_config['action_dispatch.show_exceptions']          = enabled ? :all : org_show_exceptions
```

Comment:
This should not fall back to the `org_show_exceptions`, but rather it should be some sort of feature detection. You can see the actual fix here: https://github.com/yuki24/rambulance/pull/75/files#diff-2ab79f2a64326c3e9e254b6f1494dde94ef02ef753c9777a620fc845397aaa8aL31-R36